packages/react-native/ios/DashXAppDelegate.swift

```diff
@@ -3,33 +3,36 @@ import Foundation
 @objc(DashXAppDelegate)
 class DashXAppDelegate: NSObject {
     static func swizzleDidReceiveRemoteNotification() {
-        DispatchQueue.main.async {
-            let appDelegate = UIApplication.shared.delegate
-            let appDelegateClass = object_getClass(appDelegate)
+        let appDelegate = UIApplication.shared.delegate
+        let appDelegateClass = object_getClass(appDelegate)
 
-            let originalSelector = #selector(UIApplicationDelegate.application(_:didReceiveRemoteNotification:))
```

Comment:
Due to deprecation https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623117-application